CHANGELOG.md

```diff
@@ -2,9 +2,26 @@
 
 ## Table of Contents
 
-- **[r1.2](#r12)**
+- **[r2.1](#r21)**
+- **[r1.2 - public release](#r12-public-release)**
 - **[r1.1](#r11)**
```

Comment:
If r1.2 is marked as "public release", shouldn't r2.1 then marked as "rc"? (my preference would be to only use the release numbers here).